src/i_log.c

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright(C) 2016 Adar Arnon
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+//
+// DESCRIPTION:
+//     Log interface, to avoid using printf() etc. in the code.
+//
+
+#include "SDL.h"
+
+#include <stdio.h>
+
+#include "i_log.h"
+
+
+// Prints to stdout/stderr, depending on the message priority
+void DefaultLogOuputFunction(void *userdata, int category, SDL_LogPriority priority, const char *message)
+{
+    if (priority <= SDL_LOG_PRIORITY_INFO)
+    {
+        fprintf(stdout, "%s", message);
+    }
+    else
+    {
+        fprintf(stderr, "%s", message);
+    }
+}
+
+
+void I_LogInit(void)
```

Comment:
`I_InitLog()` is more consistent with the init function names for other I_ subsystems.